pkg/arvo/tests/sys/zuse/html.hoon

```diff
@@ -89,4 +90,745 @@
+        `json`[%a ~[[%n '0e+1']]]
+    ==
+    :*  "y_number_0e1"
+        '[0e1]'
+        `json`[%a ~[[%n '0e1']]]
+    ==
+    :*  "y_number_after_space"
+        '[ 4]'
+        `json`[%a ~[[%n '4']]]
+    ==
+    :*  "y_number_double_close_to_zero"
+        '[-0.000000000000000000000000000000000000000000000000000000000000000000000000000001]'
+        `json`[%a ~[[%n '-0.000000000000000000000000000000000000000000000000000000000000000000000000000001']]]
+    ==
+    :*  "y_number_int_with_exp"
+        '[20e1]'
+        `json`[%a ~[[%n '20e1']]]
+    ==
+    :*  "y_number_minus_zero"
+        '[-0]'
+        `json`[%a ~[[%n '-0']]]
+    ==
+    :*  "y_number_negative_int"
+        '[-123]'
+        `json`[%a ~[[%n '-123']]]
+    ==
+    :*  "y_number_negative_one"
+        '[-1]'
+        `json`[%a ~[[%n '-1']]]
+    ==
+    :*  "y_number_negative_zero"
+        '[-0]'
+        `json`[%a ~[[%n '-0']]]
+    ==
+    :*  "y_number_real_capital_e"
+        '[1E22]'
+        `json`[%a ~[[%n '1E22']]]
+    ==
+    :*  "y_number_real_capital_e_neg_exp"
+        '[1E-2]'
+        `json`[%a ~[[%n '1E-2']]]
+    ==
+    :*  "y_number_real_capital_e_pos_exp"
+        '[1E+2]'
+        `json`[%a ~[[%n '1E+2']]]
+    ==
+    :*  "y_number_real_exponent"
+        '[123e45]'
+        `json`[%a ~[[%n '123e45']]]
+    ==
+    :*  "y_number_real_fraction_exponent"
+        '[123.456e78]'
+        `json`[%a ~[[%n '123.456e78']]]
+    ==
+    :*  "y_number_real_neg_exp"
+        '[1e-2]'
+        `json`[%a ~[[%n '1e-2']]]
+    ==
+    :*  "y_number_real_pos_exponent"
+        '[1e+2]'
+        `json`[%a ~[[%n '1e+2']]]
+    ==
+    :*  "y_number_simple_int"
+        '[123]'
+        `json`[%a ~[[%n '123']]]
+    ==
+    :*  "y_number_simple_real"
+        '[123.456789]'
+        `json`[%a ~[[%n '123.456789']]]
+    ==
+    :*  "y_object"
+        '{"asd":"sdf", "dfg":"fgh"}'
+        `json`(pairs ~[['asd' [%s 'sdf']] ['dfg' [%s ['fgh']]]])
+    ==
+    :*  "y_object_basic"
+        '{"asd":"sdf"}'
+        `json`(frond ['asd' [%s 'sdf']])
+    ==
+    ::  duplicated keys, it takes the latest one.
+    :*  "y_object_duplicated_key"
+        '{"a":"b","a":"c"}'
+        `json`(frond ['a' [%s 'c']])
+    ==
+    :*  "y_object_duplicated_key_and_value"
+        '{"a":"b","a":"b"}'
+        `json`(frond ['a' [%s 'b']])
+    ==
+    :*  "y_object_empty"
+        '{}'
+        `json`[%o ~]
+    ==
+    :*  "y_object_empty_key"
+        '{"":0}'
+        `json`(frond ['' [%n '0']])
+    ==
+    :*  "y_object_extreme_numbers"
+        '{ "min": -1.0e+28, "max": 1.0e+28 }'
+        `json`(pairs ~[['min' [%n '-1.0e+28']] ['max' [%n '1.0e+28']]])
+    ==
+    :*  "y_object_long_strings"
+        '{"x":[{"id": "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx"}], "id": "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx"}'
+        `json`(pairs ~[['id' [%s 'xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx']] ['x' [%a ~[(frond ['id' [%s 'xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx']])]]]])
```

Comment:
It'd be nice to keep these lines to 80 columns.

Without indentation, this would work here:

```
%:  pairs
  ['id' [%s 'xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx']]
  ['x' [%a ~[(frond ['id' [%s 'xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx']])]]]
  ~
==
```

With the indentation, you might need to use tall-form `:-` and `:~` runes. Or you could factor out the `'xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx'` cord into a variable.